### PR TITLE
Fixed nullref exception after stopping the KcpServer

### PR DESCRIPTION
**Pull request by alex89200**
After calling KcpServer.Stop() method it destroys underlying socket, but don't clear the connections list. At the next Tick those connections tries to send a "Disconnect" message and throws exceptions because socket doesn't exist anymore.
Moreover, if you start the server again - those connections remain in the list and will spam errors trying to Disconnect themselves and failing due to wrong socket binding.

In short. We need to clear existing connections when stoppping the server.

### CODE REVIEW

**Comment by vis2k** [comment]
thanks, will check this week

**Comment by vis2k** [comment]
fixed, thanks
